types/helpers/index.d.ts

```diff
@@ -0,0 +1,12 @@
+export * from './helpers.canvas';
```

Comment:
From a quick look at the source code (`src/helpers/index.js`), this is not correct, is it? Helpers aren't exported as a flat module, but rather as a single default object with several attributes.

Reply:
We have different exports for UMD and ESM.
For ESM we are exporting helpers as a directory:
https://github.com/chartjs/Chart.js/blob/da0764a585db495c18f2f450f8b3cb32a68b374e/rollup.config.js#L17-L21

Reply:
Oh, nice! I hadn't seen that.